src/test/java/com/puppycrawl/tools/checkstyle/utils/CheckUtilTest.java

```diff
@@ -443,6 +445,29 @@ public void testParseClassNames() {
             .isEqualTo(expected);
     }
 
+    @ParameterizedTest
+    @ValueSource(strings = {
+        "package-info.java",
+        "filename.package-info.java"
+    })
+    public void testInPackageInfo(String fileName) {
+        assertWithMessage("Should return true when in package info")
+            .that(CheckUtil.inPackageInfo(fileName))
+            .isTrue();
```

Comment:
can we avoid such pure unit tests ? or minimize them as much as we can

Reply:
I am afraid that if we will extend list of tests for util package, it may slow down pitest job. So such fast unit tests help to avoid it

Reply:
Not done.

Reply:
done